src/stackanalysismodule.ts

```diff
@@ -68,14 +67,18 @@ export module stackanalysismodule {
               const options = {};
               options['uri'] = `${
                 Apiendpoint.STACK_API_URL
-              }stack-analyses/${respId}?user_key=${
+                }stack-analyses/${respId}?user_key=${
                 Apiendpoint.STACK_API_USER_KEY
-              }`;
+                }`;
+              options['headers'] = {
+                'x-3scale-account-secret': 'not-set'
+              };
```

Comment:
This must be a local change, don't add it for review.

This is no longer needed if you are working with staging